src/map/utils/diggingutils.h

```diff
@@ -0,0 +1,90 @@
+/*
+===========================================================================
+
+  Copyright (c) 2010-2015 Darkstar Dev Teams
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see http://www.gnu.org/licenses/
+
+  This file is part of DarkStar-server source code.
+
+===========================================================================
+*/
+
+#ifndef _DIGGINGUTILS_H
+#define _DIGGINGUTILS_H
+
+#include <vector>
+
+#include "../../common/cbasetypes.h"
+
+/************************************************************************
+*																		*
+*	Enumerators Digging Requirements									*
+*																		*
+************************************************************************/
+
+enum REQUIREMENTS
```

Comment:
change the name to make this less ambiguous (remember that these enums are in global space, so it should be obvious what the type is referring to - for example, spell requirements use SPELLREQ)